src/config.py

```diff
@@ -14,5 +18,15 @@ def update_config():
   with open('config/2019.json', 'r') as config_file:
     config_json['2019'] = json.load(config_file)
 
+    for contributor in config_json['2019']['contributors'].items():
```

Comment:
To avoid all the `contributor[1]`s let's unpack the key and value
```suggestion
    for contributor_id, contributor in config_json['2019']['contributors'].items():
```